subsys/bluetooth/audio/ascs.c

```diff
@@ -51,6 +51,10 @@ struct bt_ascs {
 	uint8_t id;
 	bt_addr_le_t peer;
 	struct bt_ascs_ase ases[ASE_COUNT];
+	/* A single iso_channel may be used for 1 or 2 ases.
+	 * Controlled by the client.
+	 */
+	struct bt_audio_iso isos[ASE_COUNT];
```

Comment:
Why the ISO is not a member of `bt_ascs_ase`?

Reply:
That is also an option. The reason why it isn't, is that two ASE may use the same CIS and thus the same `struct bt_audio_iso`. In other words, there isn't a 1:1 between ASE and `struct bt_audio_iso`, and `struct bt_audio_iso isos[ASE_COUNT];` may also in the future to be less than `ASE_COUNT` (e.g. if we want to support fewer started streams than ASEs).

Reply:
Indeed. One ASE can be assigned to only one CIS, but one CIS can be assigned to multiple ASE's. Thus it's intuitive that each ASE keeps assigned CIS

Reply:
Not sure I follow your comment. 

If we move the ISO into the `bt_ascs_ase` then we cannot have 2 ASEs with the same ISO.

Reply:
Now I see it. I thought it's pointer rather than value. Sorry :)